src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -838,26 +834,6 @@ GetModifyConnections(Task *task, bool markCritical, bool noNewTransactions)
 		multiConnection = GetPlacementListConnection(connectionFlags, placementAccessList,
 													 NULL);
 
-		/*
```

Comment:
@marcocitus was wondering - on a call maybe? - whether we should force 2PC to on for transactions that span multiple nodes. What do you guys think?